docs/running-locally.md

```diff
@@ -153,6 +160,12 @@ Find the test that you want to run in `test/e2e`
 make TestArtifactServer  
 ```
 
+If you wish to include tests against Azure Storage, define `AUZRE=true`:
```

Comment:
Misspell here

Reply:
Thanks for the review! I’m off tomorrow, but will get these changes made early next week.

Reply:
Fixed with my latest changes.